packages/shared/src/components/CustomFeedEmptyScreen.tsx

```diff
@@ -19,11 +19,18 @@ import {
 } from './typography/Typography';
 import { LogEvent, TargetId } from '../lib/log';
 import { Button } from './buttons/Button';
-import { usePlusSubscription } from '../hooks';
+import { useConditionalFeature, usePlusSubscription } from '../hooks';
 import { IconSize } from './Icon';
+import { featurePlusCtaCopy } from '../lib/featureManagement';
 
 export const CustomFeedEmptyScreen = (): ReactElement => {
   const { logSubscriptionEvent, isPlus } = usePlusSubscription();
+  const {
+    value: { full: plusCta },
+  } = useConditionalFeature({
+    feature: featurePlusCtaCopy,
+    shouldEvaluate: !isPlus,
+  });
```

Comment:
I think it might be a good idea to introduce a hook that manages this. So that we can also check if `isAuthReady` is true as this can evaluate while the boot is being loaded.

Reply:
I can add a hook dedicated to this label but originally didn't want to over engineering this feature. 

About the evaluation: I've checked the logic in hook and once `shouldEvaluate` value change it should rerun the query and check again  (everything seems managed with useQuery and **shouldEvaluate** is both in **queryKey** and **enabled**).

Reply:
Yeah, what I mean is at on load, the value of `isPlus` is false then it would make the `shouldEvaluate` to true while the boot is still being fetched.

Reply:
I don't think having the hook would over-engineer the feature, especially since experiments are quite a mess to manage, and having a hook would make it easier to make changes across implementations.

Reply:
If you are confident that the edge case won't unnecessarily enroll users (since experiments are expensive to run), I'll trust your judgement so we can move on to merge the PR.